### ISSUE TITLE
Sodium Module says it's not loading when it is loaded in PHP

### ISSUE THREAD

**Issue by worksmarter** [nextcloud/server#30293]
![sodium_nc](https://user-images.githubusercontent.com/8888460/143051014-df3d54f5-24e7-40cb-8879-fe0c4a204016.png)

See my: `<?php phpinfo()?>`

![sodium_php](https://user-images.githubusercontent.com/8888460/143051130-e3953592-de7a-4d02-b616-1e062c213b6e.png)
What am I missing?

_Originally posted by @worksmarter in https://github.com/nextcloud/server/issues/29757#issuecomment-976657920_

**Comment by worksmarter** [comment]
## Server configuration detail

**Operating system:** Linux 3.10.0-1160.49.1.el7.x86_64 #1 SMP Tue Nov 30 15:51:32 UTC 2021 x86_64

**Webserver:** Apache/2.4.51 (Unix) OpenSSL/1.0.2k-fips (fpm-fcgi)

**Database:** mysql 10.2.41

**PHP version:** 8.0.13

Modules loaded: Core, date, libxml, openssl, pcre, sqlite3, zlib, bcmath, bz2, calendar, ctype, curl, dom, hash, fileinfo, filter, ftp, gd, gettext, gmp, SPL, iconv, session, intl, json, ldap, mbstring, standard, mysqlnd, pcntl, mysqli, PDO, pdo_mysql, pdo_sqlite, Phar, posix, pspell, Reflection, imap, shmop, SimpleXML, soap, sockets, exif, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, xml, xmlreader, xmlwriter, xsl, zip, cgi-fcgi, apcu, brotli, imagick, mailparse, mcrypt, memcache, memcached, redis, sodium, ssh2, uploadprogress, Zend OPcache

**Nextcloud version:** 22.2.3 - 22.2.3.0

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.8.0
 - activity: 2.15.0
 - apporder: 0.13.0
 - bruteforcesettings: 2.3.0
 - calendar: 3.0.2
 - camerarawpreviews: 0.7.15
 - checksum: 1.1.3
 - circles: 22.1.1
 - cloud_federation_api: 1.5.0
 - comments: 1.12.0
 - contacts: 4.0.6
 - contactsinteraction: 1.3.0
 - dashboard: 7.2.0
 - dav: 1.19.0
 - deck: 1.5.5
 - external: 3.9.0
 - federatedfilesharing: 1.12.0
 - federation: 1.12.0
 - files: 1.17.0
 - files_markdown: 2.3.5
 - files_pdfviewer: 2.3.1
 - files_rightclick: 1.1.0
 - files_sharing: 1.14.0
 - files_trashbin: 1.12.0
 - files_versions: 1.15.0
 - files_videoplayer: 1.11.0
 - firstrunwizard: 2.11.0
 - groupfolders: 10.0.0
 - logreader: 2.7.0
 - lookup_server_connector: 1.10.0
 - maps: 0.1.9
 - music: 1.5.0
 - nextcloud_announcements: 1.11.0
 - notes: 4.2.0
 - notifications: 2.10.1
 - oauth2: 1.10.0
 - password_policy: 1.12.0
 - photos: 1.4.0
 - privacy: 1.6.0
 - provisioning_api: 1.12.0
 - rainloop: 7.1.2
 - recommendations: 1.1.0
 - serverinfo: 1.12.0
 - settings: 1.4.0
 - spreed: 12.2.2
 - support: 1.5.0
 - survey_client: 1.10.0
 - tasks: 0.14.2
 - text: 3.3.0
 - theming: 1.13.0
 - twofactor_backupcodes: 1.11.0
 - updatenotification: 1.12.0
 - user_status: 1.2.0
 - viewer: 1.6.0
 - weather_status: 1.2.0
 - workflowengine: 2.4.0
Disabled:
 - admin_audit
 - encryption
 - files_external
 - files_trackdownloads
 - mail
 - richdocuments
 - sharebymail
 - systemtags
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "cloud.mydomain.com"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "22.2.3.0",
    "overwrite.cli.url": "https:\/\/vapor.userveit.com",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "integrity.check.disabled": false,
    "memcache.local": "\\OC\\Memcache\\APCu",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 6379
    },
    "theme": "",
    "maintenance": false,
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "sendmail",
    "mail_sendmailmode": "pipe",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "25",
    "default_phone_region": "US",
    "loglevel": 3,
    "logdateformat": "F d, Y H:i:s",
    "logtimezone": "America\/New_York",
    "has_internet_connection": true,
    "connectivity_check_domains": [
        "www.nextcloud.com",
        "www.startpage.com",
        "www.eff.org",
        "www.edri.org"
    ],
    "updater.release.channel": "stable",
    "app_install_overwrite": [
        "rainloop"
    ]
}
```
</details>

**External storages:** files_external is disabled

**Encryption:** no

**User-backends:** 
 * OC\User\Database


**Talk configuration:** 

STUN servers
 * stun.nextcloud.com:443

TURN servers
 * no custom server configured

Signaling servers (mode: internal):
 * no custom server configured


**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36


Under PHP Modules Loaded, Nextcloud is not indicating it is loading sodium, but it should be. This must be a bug of some sort in Nextcloud.

**Comment by acsfer** [comment]
Opening a new issue won't solve it more quickly. 😉 
Marking as duplicate of https://github.com/nextcloud/server/issues/28398 and as it still open, let's keep track of it there.

**Comment by worksmarter** [comment]
I'm sorry I didn't know that issue number was open this was a duplicate of a closed issue #29757... Thanks for the heads up.
